src/main/java/com/vistatec/ocelot/TextContextMenu.java

```diff
@@ -0,0 +1,41 @@
+package com.vistatec.ocelot;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JMenuItem;
+import javax.swing.JPopupMenu;
+
+import com.vistatec.ocelot.events.ConcordanceSearchEvent;
+import com.vistatec.ocelot.events.api.OcelotEventQueue;
+
+public class TextContextMenu extends JPopupMenu implements ActionListener {
+
```

Comment:
I'm not able to get this context menu to appear - it looks like it's supposed to be available on source segments, but it doesn't work for me.  I haven't had a chance to debug what's going on.  If this is source-only, the name of this class could be more clear.